### DISCUSSION TITLE
YDP03X可以强刷YDP03XS的词典笔OS吗

### DISCUSSION THREAD

**Discussion by SkySight-666** [PenUniverse/PenMods#218] [你问我答]
我同学想要网易云
但是YDP03X官方适配似乎遥遥无望。
如题，我发现固件区已经有固件，这要注意些什么吗（均为16g版本）

**Comment by JimmyGuo12** [comment]
在固件里面把parameter.txt的MACHINE改成3X的MACHINE号

**Reply by SkySight-666** [reply]
可是b站评论区有人反馈会无法激活，这样做可以解决这个问题吗

**Reply by gaopx11233** [reply]
试过了，无法激活

**Reply by SkySight-666** [reply]
是的，但是discussions中已经有3x的rom了，直接刷入就好了
https://github.com/PenUniverse/PenMods-release/discussions/239
